hack/prime_test_cluster.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
 # cleanup
-rm -fr hack/rootCA.pem
+if [[ -f hack/rootCA.pem ]]; then
```

Comment:
Why do you need the if?

Reply:
because `-f` (force-)pushes are not best practice in scripts. It is surely opinionated, but force-removals can be risky and are not necessary in most situations.

Reply:
In this situation we also want to make sure that a rootCA with a bad chmod or a different user is also removed. Or if it was created with a super user. But meh. :) I'm not gonna die on this hill. :D